src/js/angular/ttyg/templates/chat-panel.html

```diff
@@ -35,12 +38,15 @@
                         <i class="fa-regular fa-wand-magic-sparkles"></i>
                     </div>
                 </div>
+                <div class="assistant-progress" ng-if="!chatItem.answer">
```

Comment:
This `assistant-progress` should be like `answer-loading` or `answer-progress`

Reply:
done